Quaver.Shared/Screens/Settings/SettingsDialog.cs

```diff
@@ -325,7 +325,8 @@ private void CreateSections()
                     new SettingsBool(this, "Display Song Time Progress", ConfigManager.DisplaySongTimeProgress),
                     new SettingsBool(this, "Animate Judgement Counter", ConfigManager.AnimateJudgementCounter),
                     new SettingsBool(this, "Display Scoreboard", ConfigManager.ScoreboardVisible),
-                    new SettingsBool(this, "Tap to Pause", ConfigManager.TapToPause)
+                    new SettingsBool(this, "Tap to Pause", ConfigManager.TapToPause),
+                    new SettingsBool(this, "Display Failed Scores", ConfigManager.DisplayFailedScores)
```

Comment:
Place this in the Miscellaneous section and also name it `Display Failed Local Scores`. Rename the property as well to `DisplayFailedLocalScores`

Reply:
aight